src/server/server_family.cc

```diff
@@ -935,13 +934,40 @@ struct AggregateLoadResult {
   std::atomic<size_t> keys_read;
 };
 
+void ServerFamily::FlushAll(ConnectionContext* cntx) {
+  const CommandId* cid = service_.FindCmd("FLUSHALL");
+  boost::intrusive_ptr<Transaction> flush_trans(new Transaction{cid});
+  flush_trans->InitByArgs(cntx->ns, 0, {});
+  VLOG(1) << "Performing flush";
+  error_code ec = Drakarys(flush_trans.get(), DbSlice::kDbAll);
+  if (ec) {
+    LOG(ERROR) << "Error flushing db " << ec.message();
+  }
+}
+
 // Load starts as many fibers as there are files to load each one separately.
 // It starts one more fiber that waits for all load fibers to finish and returns the first
 // error (if any occured) with a future.
-std::optional<fb2::Future<GenericError>> ServerFamily::Load(const std::string& load_path) {
+std::optional<fb2::Future<GenericError>> ServerFamily::Load(string_view load_path,
+                                                            LoadExistingKeys existing_keys) {
+  fs::path path(load_path);
+
+  if (load_path.empty()) {
+    fs::path dir_path(GetFlag(FLAGS_dir));
+    string filename = GetFlag(FLAGS_dbfilename);
+    dir_path.append(filename);
+    path = dir_path;
```

Comment:
I think it should work like this "dir_path / filename;"

Reply:
This code is moved from debugcmd, I didn't touch it :)
But using `fs::path` is better than appending strings for various reasons

Reply:
I suggest to use fs::path syntax instead of append you can use / operator for path concatenation